### ISSUE TITLE
custom android sqlite path,sqlitestudio cant find

### ISSUE THREAD

**Issue by shaojiankui** [pawelsalawa/sqlitestudio#3411]
when i use sqlitestudio for android。
but android sqlite is create by dbhepler with full path not name.
` this(context, fullpathwithname, null,Version);`
so the sqlite file not exist in “/data/data/appid/databases”,
it may be create at /sdcard or some custom path.
the sqlitestudio app cant find this database.
![c84a1253-248e-47fa-b5a0-85d11ec4832b](https://user-images.githubusercontent.com/5194500/44073011-5960f62e-9fc4-11e8-8d34-b9fae23ed778.png)

**Comment by shaojiankui** [comment]
Override the "SQLiteStudioService.instance().start" argument context databaseList() method
```
public class APP extends Application {
    @Override
    public String[] databaseList() {
        String[] databaseList = super.databaseList();
        String[] temp = new String[databaseList.length + 1];
        for (int i = 0; i < databaseList.length; i++) {
            temp[i] = databaseList[i];
        }
        temp[databaseList.length] = FileManager.getRootFilePath() + "pss.db";
        return temp;
    }
    @Override
    public void onCreate() {
        super.onCreate();
        SQLiteStudioService.instance().start(this);
    }
```

**Comment by pawelsalawa** [comment]
This is no longer relevant, as Android plugin is now decomissioned. There are tools build into Android's IDEs that serve similar purpose, but likely better.

**Comment by pawelsalawa** [comment]
Reopened in attempt to still maintain the plugin.

**Comment by pawelsalawa** [comment]
This should be done after #3968
